packages/cli-repl/test/e2e.spec.ts

```diff
@@ -55,6 +55,41 @@ describe('e2e', function() {
       });
     });
 
+    it.only('throws multiline input with a single line string', async () => {
+      // this is an unterminated string constant and should throw, since it does
+      // not pass: https://www.ecma-international.org/ecma-262/#sec-line-terminators
+      shell.stdio.stdin.write('"this is a multi\nline string"\n');
```

Comment:
@mcasimir I added some e2e tests, but the test you suggested is actually meant to not be recoverable as per the ecma spec in the comment there. I added some other ones that are meant to be recovered.

Reply:
sounds good!